README.md

```diff
@@ -3,25 +3,35 @@
 ## Requirements
 
 - MkDocs
-- Mkdocs Material theme
+- MkDocs Material theme
 
-## Build locally
+## Build
 
 Clone repo and navigate to directory:
 
 ```bash
-git clone https://github.com/mimblewimble/docs.git
-cd docs
+git clone --recursive https://github.com/mimblewimble/docs.git
 ```
 
+### Local
+
 Start the local server:
 
 ```bash
 mkdocs serve
 ```
 
-Visit URL to see live changes:
+### Container
+
+Start the local server:
+
+```bash
+docker build -t docs -f Containerfile .
+docker run -p 8000:8000 -v ./:/app/:ro docs:latest
```

Comment:
That is something I commonly use across different projects I worked on. It might be a personal preference but the name Dockerfile is not technology agnostic. There are many other tools out there that can be used to build and run container images such as `podman`, `rkt` e.g. but there is no such thing as a Rktfile or similar. There are just OCI compliant container images and those that are not hence my effort to avoid the term `docker` as much as possible.

Reply:
Okay, to be honest I have no strong preferences regarding the particular naming of the file. So let's consider it solved.